README.md

```diff
@@ -17,6 +17,7 @@ Browser automation is the act of executing actions automatically in a web browse
 * [Alumnium](https://alumnium.ai) - Open-source AI-powered test automation library on top of Playwright/Selenium.
 * [Axiom](https://axiom.ai) - No code browser automation tool, like Zapier.
 * [Browserflow](https://browserflow.app) - Chrome extension to automate your local browser or in the cloud.
+* [Browser-Use](https://github.com/browser-use/browser-use) - Python library and service to automate browsing using AI agents and CDP.
```

Comment:
Hi, I think this is a good fit for this list, just a small change on the description, I assume `CDP` stands for `Chrome DevTools Protocol`, please state that explicitly, as the initials may not be obvious to a reader:

```suggestion
* [Browser-Use](https://github.com/browser-use/browser-use) - Python library and service to automate browsing using AI agents and Chrome DevTools Protocol.
```